ProjFS.Mac/PrjFSKext/PrjFSKext/KauthHandler.cpp

```diff
@@ -288,6 +289,22 @@ static int HandleVnodeOperation(
         goto CleanupAndReturn;
     }
     
+    if (ActionBitIsSet(action, KAUTH_VNODE_DELETE))
+    {
+        if (!TrySendRequestAndWaitForResponse(
+                root,
+                VDIR == vnodeType ? MessageType_KtoU_NotifyDirectoryPreDelete : MessageType_KtoU_NotifyFilePreDelete,
```

Comment:
This TrySendRequestAndWaitForResponse and the one below structure their ternary statements differently. I have a preference for how I think it should look, but you can ignore that so long as they're consistent with each other :)

Can we pull the comparison up into a separate variable here too? Additionally, this line is getting long, I'd rather we break it up into 3 separate lines like we do in the below addition.

Reply:
> This TrySendRequestAndWaitForResponse and the one below structure their ternary statements differently. I have a preference for how I think it should look, but you can ignore that so long as they're consistent with each other

I can make them consistent, but could you provide more details on how you see them as inconsistent?

> Can we pull the comparison up into a separate variable here too? Additionally, this line is getting long, I'd rather we break it up into 3 separate lines like we do in the below addition.

Yes to both of these, I'll make the change.

Reply:
@nickgra I think I've addressed this feedback with my latest commit, but let me know if there's still changes you'd like to see.